scripts/apps/authoring/authoring/controllers/ChangeImageController.ts

```diff
@@ -217,6 +217,16 @@ export function ChangeImageController($scope, notify, _, api, $rootScope, deploy
     * @description
     */
     $scope.cancelMetadataChanges = () => {
+        const selectedUsageTerms = [];
```

Comment:
The function which is canceling changes shouldn't need to know about specific fields. Changes should be kept in one object and resetting that object should be enough. Otherwise, it will continuously cause bugs as fields change.